requirements.txt

```diff
@@ -46,8 +46,9 @@ Rx==1.6.1
 singledispatch==3.7.0
 six==1.16.0
 soupsieve==2.3.2.post1
-sqlparse==0.4.4
+sqlparse==0.5.0
 text-unidecode==1.3
+typing_extensions==4.11.0
```

Comment:
This one is brand new.  I just want to confirm that it wasn't added accidentally?

Reply:
I noticed that too, and tried to remove it, but it seems to be added back as a dependency to asgiref:
```
Collecting typing-extensions>=4 (from asgiref==3.8.1->-r requirements.txt (line 2))
  Using cached typing_extensions-4.11.0-py3-none-any.whl.metadata (3.0 kB)
```